src/commands/queue.rs

```diff
@@ -0,0 +1,32 @@
+use std::sync::Arc;
+
+use anyhow::Result;
+use serenity::async_trait;
+use serenity::builder::CreateApplicationCommand;
+
+use crate::{command_context::CommandContext, store::guild_action::print_queue::PrintQueue};
+
+use super::{JoovyCommand, JoovyCommands};
+
+#[derive(Default, Debug)]
+pub struct Queue;
+
+#[async_trait]
+impl JoovyCommand for Queue {
+    fn create_application_command<'a>(
+        &self,
+        command: &'a mut CreateApplicationCommand,
+    ) -> &'a mut CreateApplicationCommand {
+        command
+            .name(JoovyCommands::Queue(Queue).as_ref())
+            .description("Displays the track queue")
```

Comment:
Kan nog ta bort s:et

Reply:
Kanske kan skippa "track" också. Vad sägs som "Display the queue"?

Reply:
Yea, låter bra 👍